src/ui/components/CheckboxOneEdt/CheckboxOneEdt.tsx

```diff
@@ -123,7 +123,7 @@ const CheckboxOneEdt = memo((props: CheckboxOneProps) => {
                     {label && (
                         <Box>
                             <Box className={classes.labelSpacer}>
-                                <label>{label}&nbsp;?</label>
+                                <h1 className={classes.h1}>{label}&nbsp;?</h1>
```

Comment:
Explain use of main title here (=>h1)

Reply:
we have changed it due to two types of accessibility bugs.

We cannot have a label without being associated to a component by default + we must have a title per page.

As we currently have pages where we directly associate the page to these components, we have changed the labels (indirectly we had already conceived the question as a title) for titles.

Is there any problem for you that we have made this change?

Reply:
No problem for me. We ll see with the expert in accessiblity. Thank you for the response.